packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -256,7 +259,9 @@ export function createFetch(
 
   if (process.env.NODE_ENV === 'production') {
     if (process.env.__NEXT_CONFIG_OUTPUT === 'export') {
-      if (fetchUrl.pathname.endsWith('/')) {
+      if (fetchUrl.pathname === basePath || skipTrailingSlashRedirect) {
```

Comment:
I'm not sure it's correct to always add if skipTrailingSlashRedirect is configured as this is more specific to where the file would be located which depends on the `trailingSlash` config https://github.com/vercel/next.js/blob/canary/packages/next/src/export/index.ts#L96

Reply:
I thought the file name was correct and the request path was incorrect, but could it actually be the other way around? That is, the request path is correct, and the file name is incorrect? The way to fix this will vary depending on the policy.